apps/galaxy/src/modules/auth/SignUpForm/CompanyInfoPage.tsx

```diff
@@ -42,58 +42,50 @@ const CompanyInfoPage = (props: {
   const goNextFromStep = () => {
     setFormStep(formStep + 1);
   };
-  const goBackFromStep = () => {
-    setFormStep(formStep - 1);
-  };
 
   return (
-    <>
-      <FormControl as="form" onSubmit={onSubmit}>
-        <Stack>
-          <FormInput
-            {...businessName}
-            {...register("businessName")}
-            errorMessage={formState.errors.businessName?.message}
-          />
-          <FormInput
-            {...legalEntityName}
-            {...register("legalEntityName")}
-            errorMessage={formState.errors.legalEntityName?.message}
-          />
-          <FormInput {...abn} {...register("abn")} errorMessage={formState.errors.abn?.message} />
-          <FormInput
-            {...contactNumber}
-            {...register("contactNumber")}
-            errorMessage={formState.errors.contactNumber?.message}
-          />
-          <FormSelect
-            {...companyState}
-            {...register("companyState")}
-            errorMessage={formState.errors.companyState?.message}
-          />
-          <FormInput
-            {...companyPostcode}
-            {...register("companyPostcode")}
-            errorMessage={formState.errors.companyPostcode?.message}
-          />
-          <FormInput
-            {...businessAddress}
-            {...register("businessAddress")}
-            errorMessage={formState.errors.businessAddress?.message}
-          />
+    <FormControl as="form" onSubmit={onSubmit}>
+      <Stack>
+        <FormInput
+          {...businessName}
+          {...register("businessName")}
+          errorMessage={formState.errors.businessName?.message}
+        />
+        <FormInput
+          {...legalEntityName}
+          {...register("legalEntityName")}
+          errorMessage={formState.errors.legalEntityName?.message}
+        />
+        <FormInput {...abn} {...register("abn")} errorMessage={formState.errors.abn?.message} />
+        <FormInput
+          {...contactNumber}
+          {...register("contactNumber")}
+          errorMessage={formState.errors.contactNumber?.message}
+        />
+        <FormSelect
+          {...companyState}
+          {...register("companyState")}
+          errorMessage={formState.errors.companyState?.message}
+        />
+        <FormInput
+          {...companyPostcode}
+          {...register("companyPostcode")}
+          errorMessage={formState.errors.companyPostcode?.message}
+        />
+        <FormInput
+          {...businessAddress}
+          {...register("businessAddress")}
+          errorMessage={formState.errors.businessAddress?.message}
+        />
+      </Stack>
+      <Flex direction="column" gap="16px">
+        <Stack marginTop="24px" direction={["column", "row"]} justifyContent="stretch">
```

Comment:
Do you really need `<Stack>` here if there is only one button now?

Reply:
I think keeping <Stack> here would be fine. The reason for this is that it is more appropriate to use a global layout than to hard code the height of the button in the component.
![image](https://user-images.githubusercontent.com/73293255/200094234-f948ed5d-24ca-4b89-be2c-cd6b51fc8b5b.png)
![image](https://user-images.githubusercontent.com/73293255/200094715-2e21d87c-e27d-4b00-be93-2343a2dcfb12.png)

Also, I removed the direction property as there is now only one button.

Reply:
I don't see how this is related to `<GlobalLyaout>` and it is fine to have a or a set of fixed height for the buttons.